packages/gafl-webapp-service/src/processors/concession-helper.js

```diff
@@ -0,0 +1,105 @@
+import { CONCESSION, CONCESSION_PROOF } from './mapping-constants.js'
```

Comment:
It looks like each of the addXXX methods here always remove the other type of concessions?  If so, would it not be simpler to just recreate the array each time?

Reply:
It would be I agree but the thought was that this should basically cover the situation where there are multiple concessions without to much change